crates/accelerate/src/sparse_observable.rs

```diff
@@ -110,7 +90,7 @@ impl BitTerm {
     /// Get the label of this `BitTerm` used in Python-space applications.  This is a single-letter
     /// string.
     #[inline]
-    fn py_label(&self) -> &'static str {
+    pub(crate) fn py_label(&self) -> &'static str {
```

Comment:
If these are purely for Python consumption and unrelated to the Rust code (which I believe is the case), perhaps consider moving them to be regular functions that live with the rest of the Python-specific code?

Reply:
Also fine, yeah. I guess it's important to note that the actual values here are part of the Python interface and need to be stable.